protocol/src/main/resources/protocol.xml

```diff
@@ -140,6 +140,7 @@
     <!-- populated when RecordType is COMMAND_REJECTION -->
     <field name="rejectionType" id="7" type="RejectionType"/>
     <field name="brokerVersion" id="9" type="Version" sinceVersion="2" presence="optional"/>
+    <field name="recordVersion" id="10" type="uint16" sinceVersion="3" presence="optional"/>
```

Comment:
🔧 We should move this up a few lines. It's now under `<!-- populated when RecordType is COMMAND_REJECTION -->`. But this is also filled for records that aren't command rejections. I'd argue the line above should also be moved up tbh.